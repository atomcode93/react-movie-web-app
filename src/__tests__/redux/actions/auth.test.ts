import { actions as authActions } from 'redux/ducks/auth.duck';

describe('Auth Actions', () => {
  test('should set user', () => {
    const action = authActions.setUserSuccess({
      type: 'SET_USER_SUCCESS',
      payload: {
        user: {},
      },
    });
    expect(action).toMatchSnapshot();
  });
  test('should set initial state on authLogout', () => {
    const action = authActions.authLogout();
    expect(action).toMatchSnapshot();
  });
});
