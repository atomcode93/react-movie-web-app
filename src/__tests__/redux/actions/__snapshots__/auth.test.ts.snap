// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth Actions should set initial state on authLogout 1`] = `
Object {
  "type": "AUTH_LOGOUT",
}
`;

exports[`Auth Actions should set user 1`] = `
Object {
  "payload": Object {
    "payload": Object {
      "user": Object {},
    },
    "type": "SET_USER_SUCCESS",
  },
  "type": "SET_USER_SUCCESS",
}
`;
